setup.py

```diff
@@ -24,7 +24,7 @@
 
 setup(
     name="tempoiq",
-    version="1.0.1",
+    version="1.0.2",
```

Comment:
Our current policy is to hold off on bumping version number until after something is merged.